ReactWindows/ReactNative/Views/Text/ReactTextViewManager.cs

```diff
@@ -1,4 +1,6 @@
-﻿using ReactNative.UIManager;
+using ReactNative.Common;
```

Comment:
>u [](start = 0, length = 1)

Can you describe what scenarios we want to support in our text implementation? For example, does `importantForAccessibility` work on nested text components or only on the topmost Text?

Reply:
Any combination of nested elements should work properly provided that elements implement importantForAccesibility. I will include this in the checkin description. 

I talked to Yuri about a document that we can put in docs. We are trying to implement what is already stated in ReactXP/ReactNative accessibility documentation but we can provide some extra details for the specifics of implementation.

---
In reply to: [164944309](https://github.com/Microsoft/react-native-windows/pull/1595#discussion_r164944309) [](ancestors = 164944309)

Reply:
Discussed in person. We'll provide an update for approval by FB with details to "react-native" accessibility document based on inferred behavior from Android and iOS implementation. We also want to provide an "accessibility" document in "react-native-windows" that will contain implementation details and will facilitate future enhancements to "react-native-windows" accessibility implementation.

---
In reply to: [165803907](https://github.com/Microsoft/react-native-windows/pull/1595#discussion_r165803907) [](ancestors = 165803907,165528024,164944309)